deploy/flux-deployment.yaml

```diff
@@ -2,12 +2,12 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: flux
+  app: flux
```

Comment:
I don't think `app` is a valid metadata. See: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#objectmeta-v1-meta
You might be looking to use:
```
metadata:
  name: flux
  labels:
    app: flux
```

Reply:
Right you are -- good eye.  The label in the deployment template the only thing I needed to change.